### ISSUE TITLE
decide upon the appropriate formatting for footnotes

### ISSUE THREAD

**Issue by nicholasjhorton** [STAT325-S24/HistoryAmherstCollege#6]
Footnotes are common in the text. See for example page 513.

<img width="318" alt="Screenshot 2024-03-12 at 11 21 04 AM" src="https://github.com/STAT325-S24/HistoryAmherstCollege/assets/530542/602a8be0-0ea7-4227-8e68-15f8ebcfa3b1">

See `Amherst College. I` and parenthetical text below

```
century sermon more than ten years ago, and is still able to
preach, has always been a warm friend of Amherst College. l
Entering College when he was only fourteen, he was a boy in
stature and a boy in his love of ease and pleasure till his Junior
year, wnen for the first time he began to awake to a genuine
love of those studies in which his strength has ever since lain,
viz., classics, rhetoric and belles-lettres. He graduated with
highly respectable standing in the Class of '39, of which Dr.
Huntington was the Valedictorian, and of which some of the
brightest ornaments, Bancroft, Miller, Palmer and others, died
within a few years after their graduation. He then taught with
marked success, for some years, in Monson Academy and Willis-
ton Seminary. He studied law also, before the great questions
of personal religion and his life-work were settled. At length  
he went to Andover where these questions were made clear 
to him, and where he completed the theological course in 1845.
The same year he took charge of the Congregational Church  
at Brookline, Mass. In November, 1846, he accepted a call to
the Church of the Pilgrims in Brooklyn, of which he is still 
pastor. Our readers need not be told that that church is the 
center, and its pastor the head-center of Congregationalism in 
" the City of Churches." Nor is his power confined to the pulpit
or within the pale of the church, still less of any denomination.
His influence is felt everywhere. The Brooklyn Historical So-
ciety, with its Library and Museum, is his foster-child. Litera-
ture, art, politics, morals, education and religion all feel his
guiding and inspiring touch.
(1 Mrs. Billings of Conway, who might well be called one of the founders, was of
the Storrs family.)
```

This issue will be closed when a plan for how to handle footnotes is proposed and agreed upon.

**Comment by jpapagelis24** [comment]
Our proposal is to extract the footnote sections from the pages and then paste them into the section they correspond to enclosed in square brackets. This would have to be done manually to some extent. The above page would look like:

```
century sermon more than ten years ago, and is still able to
preach, has always been a warm friend of Amherst College. [Mrs. Billings of Conway, who might well be called one of the founders, was of the Storrs family.]
Entering College when he was only fourteen, he was a boy in
stature and a boy in his love of ease and pleasure till his Junior
year, wnen for the first time he began to awake to a genuine
love of those studies in which his strength has ever since lain,
viz., classics, rhetoric and belles-lettres. He graduated with
highly respectable standing in the Class of '39, of which Dr.
Huntington was the Valedictorian, and of which some of the
brightest ornaments, Bancroft, Miller, Palmer and others, died
within a few years after their graduation. He then taught with
marked success, for some years, in Monson Academy and Willis-
ton Seminary. He studied law also, before the great questions
of personal religion and his life-work were settled. At length  
he went to Andover where these questions were made clear 
to him, and where he completed the theological course in 1845.
The same year he took charge of the Congregational Church  
at Brookline, Mass. In November, 1846, he accepted a call to
the Church of the Pilgrims in Brooklyn, of which he is still 
pastor. Our readers need not be told that that church is the 
center, and its pastor the head-center of Congregationalism in 
" the City of Churches." Nor is his power confined to the pulpit
or within the pale of the church, still less of any denomination.
His influence is felt everywhere. The Brooklyn Historical So-
ciety, with its Library and Museum, is his foster-child. Litera-
ture, art, politics, morals, education and religion all feel his
guiding and inspiring touch.
```

**Comment by nicholasjhorton** [comment]
I wanted to thank you all for your thoughtful brainstorming: it was very helpful to be able to explore various ways of tagging the footnotes.

I'm happy with @jpapagelis24's proposal that footnotes be moved into bracketed sentences:

```
... has always been a warm friend of 
Amherst College. [Mrs. Billings of Conway, who might well be 
called one of the founders, was of the Storrs family.]
```

Implementation will involve some work, but that can be deferred until after break.

**Comment by nicholasjhorton** [comment]
We need to start to document our process. This issue will be closed when our decision on the formatting of footnotes is documented in an appendix of a report. We should discuss where this might be located.

**Comment by nicholasjhorton** [comment]
Perhaps in a Wiki? https://github.com/STAT325-S24/HistoryAmherstCollege/wiki

**Comment by nicholasjhorton** [comment]
Closed in favor of #28
